diff --git a/scripts/react-native-xcode.sh b/scripts/react-native-xcode.sh
index 001b6cb7c320330c29d34e3f33931513158c9995..f04a246426f1ead3298eb25e932f24dacc259186 100755
--- a/scripts/react-native-xcode.sh
+++ b/scripts/react-native-xcode.sh
@@ -142,14 +142,10 @@ fi
 # Allow opting out of using npx react-native config
 if [[ -n "$CONFIG_JSON" ]]; then
   EXTRA_ARGS+=("--load-config" "$CONFIG_JSON")
-elif [[ -n "$CONFIG_CMD" ]]; then
-  EXTRA_ARGS+=("--config-cmd" "$CONFIG_CMD")
-else
-  EXTRA_ARGS+=("--config-cmd" "$NODE_BINARY $NODE_ARGS $REACT_NATIVE_DIR/cli.js config")
 fi
 
 # shellcheck disable=SC2086
-"$NODE_BINARY" $NODE_ARGS "$CLI_PATH" $BUNDLE_COMMAND \
+NODE_PATH="$NODE_PATH" "$NODE_BINARY" $NODE_ARGS "$CLI_PATH" $BUNDLE_COMMAND \
   $CONFIG_ARG \
   --entry-file "$ENTRY_FILE" \
   --platform "$BUNDLE_PLATFORM" \